jobs/ml-ops/container/README.md

```diff
@@ -0,0 +1,37 @@
+# Deploy an inference API as a container
+
+## Step 1: Build and push API image to Scaleway's Registry
+
+```bash
+docker build -t rg.fr-par.scw.cloud/inference-api-images/inference-api:v1 .
+docker login rg.fr-par.scw.cloud/inference-api-images -u nologin --password-stdin <<< "$SCW_SECRET_KEY"
+docker push rg.fr-par.scw.cloud/inference-api-images/inference-api:v1
+```
```

Comment:
See my comment about the Terraform setup, but basically, this can be done within the Terraform configuration. It avoids adding an extra deployment step.